frontend/src/components/paymentmodule/PaymentPlanDetails/PaymentPlanDetailsHeader/SplitIntoPaymentLists.tsx

```diff
@@ -44,17 +44,26 @@ export const SplitIntoPaymentLists = ({
   const [mutate, { loading }] = useSplitPpMutation();
   const { showMessage } = useSnackbar();
 
+  let minPaymentsNoMessage = 'Payments Number must be greater than 10';
+  let maxPaymentsNoMessage = `Payments Number must be less than ${paymentPlan.paymentItems.totalCount}`;
+
+  if (paymentPlan.paymentItems.totalCount <= 10) {
```

Comment:
please change to:
"There are too few payments (${paymentPlan.paymentItems.totalCount}) to split."